Packs/Sixgill/metadata.json

```
@@ -0,0 +1 @@
+[]
```

```suggestion
{
	"name": "Sixgill DarkFeed",
	"description": "Description of the pack:",
	"support": "partner",
	"beta": false,
	"deprecated": false,
	"serverMinVersion": "4.5.0",
	"currentVersion": "1.0.0",
	"author": "Sixgill",
	"url": "The url of support, should represent your GitHub account:",
	"email": "The email in which you can be contacted in:",
	"categories": "Data Enrichment & Threat Intelligence",
	"useCases": "",
	"keywords": ""
}
```

just fill `description` , `author`, `url` or `email` for contact details - who should we contact in case there is a bug/support case/etc.